### DISCUSSION TITLE
XAML Designer/Previewer on VS Code?

### DISCUSSION THREAD

**Discussion by martin-braun** [AvaloniaUI/Avalonia#7224] [Ideas]
I'm pretty sure that has been asked before, but I don't find any thread or something regarding this. I know that there is only a full-fledged designer for VS and a preview plugin for Rider, I just wonder if there are any plans to build a previewer / designer for VS Code at all? 

VS Code is the most loved lightweight cross-platform IDE for many reasons and I would love to experience Avalonia development with at least a live preview, but preferably with a full-fledged cross-platform designer within it.

**Comment by maxkatz6** [comment]
https://github.com/AvaloniaUI/Avalonia/issues/999

**Reply by martin-braun** [reply]
Thanks, was unable to find it.
